src/main/groovy/com/avioconsulting/mule/linter/model/RuleViolation.groovy

```diff
@@ -0,0 +1,68 @@
+package com.avioconsulting.mule.linter.model
+
+class RuleViolation {
+    Rule rule
+    Integer lineNumber
+    String fileName
+    RuleSeverity severity
```

Comment:
If a Rule has a severity, why do we need to capture both the Rule and severity here?